packages/fluentui/local-sandbox/package.json

```
@@ -15,18 +15,18 @@
   "devDependencies": {
     "@types/react": "16.8.25",
     "@types/react-dom": "16.8.4",
-    "babel-loader": "^8.0.6",
-    "copy-webpack-plugin": "^4.5.2",
+    "babel-loader": "^8.2.2",
+    "copy-webpack-plugin": "^7.0.0",
     "express": "^4.15.4",
-    "fork-ts-checker-webpack-plugin": "1.3.3",
+    "fork-ts-checker-webpack-plugin": "6.1.0",
```

```suggestion
    "fork-ts-checker-webpack-plugin": "^6.1.0",
```

nit, I don't think that an exact version is required

---

will fix, ty